src/explore-education-statistics-common/src/components/form/rhf/__tests__/RHFForm.test.tsx

```diff
@@ -0,0 +1,491 @@
+import FormProvider from '@common/components/form/rhf/FormProvider';
+import RHFForm from '@common/components/form/rhf/RHFForm';
+import Yup from '@common/validation/yup';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import noop from 'lodash/noop';
+import React from 'react';
+
+describe('Form', () => {
+  test('renders error summary from form errors when form is submitted', async () => {
+    const { container } = render(
+      <FormProvider
+        initialValues={{
+          firstName: '',
+          lastName: '',
+        }}
+        validationSchema={Yup.object({
+          firstName: Yup.string().required('First name is required'),
+          lastName: Yup.string().required('Last name is required'),
+        })}
+      >
+        {() => {
+          return (
```

Comment:
As other places - could spare some lines in this test suite by using the non-render prop variant for the children.